numpy/core/src/multiarray/datetime_strings.c

```diff
@@ -1075,7 +924,7 @@ lossless_unit_from_datetimestruct(npy_datetimestruct *dts)
  */
 NPY_NO_EXPORT int
 make_iso_8601_datetime(npy_datetimestruct *dts, char *outstr, int outlen,
-                    int local, NPY_DATETIMEUNIT base, int tzoffset,
+                    int local, int utc, NPY_DATETIMEUNIT base, int tzoffset,
```

Comment:
Probably would be good to add information about the `utc` parameter to the doc comment.